Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -411,6 +412,7 @@ void WebChromeClient::setResizable(bool resizable)
 void WebChromeClient::addMessageToConsole(MessageSource source, MessageLevel level, const String& message, unsigned lineNumber, unsigned columnNumber, const String& sourceID)
 {
     // Notify the bundle client.
+    LOG(Loading, "addMessageToConsole-> message:%s", message.utf8().data());
```

Comment:
I assume that log was useful for testing, but lets remove it now

Reply:
In some case, we are getting logs from here like on below cases:
1.http://mvt.onemw.net/test-materials/sandbox/js_errors.html
>>Nov 23 13:31:48 E0B7B1-APPSTB-301000041105 wpe.sh[5416]: ddMessageToConsole-> message:console.error - should include stack trace

2.var xhr = new XMLHttpRequest(); xhr.open('GET', 'https://example.com', true); xhr.send();
>>Nov 23 13:31:18 E0B7B1-APPSTB-301000041105 WPEWebProcess[5416]: 231123-13:31:18.029125 [mod=Loading, lvl=INFO] [tid=15] addMessageToConsole-> message:XMLHttpRequest cannot load https://example.com/ due to access control checks.

That's why added this print